src/zenml/zen_stores/migrations/versions/8a64fbfecda0_add_num_outputs_to_run_step.py

```diff
@@ -0,0 +1,45 @@
+"""Add num_outputs to run step [8a64fbfecda0].
+
+Revision ID: 8a64fbfecda0
+Revises: 5330ba58bf20
+Create Date: 2022-11-08 16:20:35.241562
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "8a64fbfecda0"
+down_revision = "5330ba58bf20"  # 0.21.1
```

Comment:
```suggestion
down_revision = "5330ba58bf20"
```

Why the comment here?